media/css/firefox/whatsnew/whatsnew-57.scss

```diff
@@ -0,0 +1,240 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
+
+//* -------------------------------------------------------------------------- */
+// Common elements
+
+body {
+    @include open-sans;
+    overflow-x: hidden;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-weight: normal;
+}
+
+#outer-wrapper {
+    overflow-x: hidden;
+}
+
+
+
+//* -------------------------------------------------------------------------- */
+// Standard page
+
+.intro {
+    background: #c7216a;
+    background: url('/media/img/firefox/whatsnew_57/bg-tail.svg') right top no-repeat,
+                linear-gradient(148deg, #c7216a 7%, #fda41e 100%);
+    background-size: cover;
+    color: #fff;
+    padding: 40px 0 140px;
+    position: relative;
+    z-index: 1;
+    overflow: hidden;
+
+    &:after {
+        @include at2x('/media/img/firefox/whatsnew_57/new-tab.jpg', 280px, 202px);
+        background-position: top right;
+        background-repeat: no-repeat;
+        bottom: -40px;
+        content: '';
+        height: 202px;
+        margin-right: -140px;
+        position: absolute;
+        right: 50%;
+        width: 280px;
+    }
+
+
+    .intro-content {
+        text-align: center;
+
+        h1 {
+            @include image-replaced;
+            background: url('/media/img/firefox/whatsnew_57/logo-wordmark.png') no-repeat;
+            @include background-size(212px 45px);
+            height: 45px;
+            margin: 0 auto 40px;
+            width: 212px;
+        }
+
+        h2 {
+            @include font-size-level3;
+            font-weight: bold;
+        }
+
+        .tagline {
+            margin: 20px 0 40px;
+
+            br {
```

Comment:
Need to remove this (CC @MikkCZ)
https://bedrock-demo-craigcook.us-west.moz.works/cs/firefox/57.0/whatsnew/

Reply:
For reference, this is really bad. You would end up removing a perfectly fine `<br>`, and breaking the sentence (e.g. missing a space after a period or comma).